static/prelogin/less/components/pricing-section.less

```diff
@@ -170,7 +170,7 @@ row heights can be found in components/pricing.less
 
   @media (max-width: @smallest-table-width) {
     overflow-x: scroll;
-    overflow-y: none;
+    overflow-y: hidden;
```

Comment:
@orangejenny Having `none` makes it scroll on y, on xs-devices. Making it hidden removes the unnecessary scroll. No any reason, why this was `none`?